lib/ExternalMap.ts

```diff
@@ -0,0 +1,55 @@
+import { showLocation } from "react-native-map-link"
+import * as Location from "expo-location"
+import {
+  LocationCoordinate2D,
+  Placemark,
+  UserLocationTrackingUpdate,
+  placemarkToFormattedAddress
+} from "./location"
+
+
+export type EventMapDetails = {
+  coordinates: LocationCoordinate2D,
+  placemark?: Placemark
+}
+
+const mapOptions = (details: EventMapDetails) => {
```

Comment:
The names could be a bit clearer. At least add a prefix like "external" so we know it's for the map app and not related to the map component